packages/layout/src/components/cards/common/views/address/addressPage.tsx

```diff
@@ -79,6 +79,7 @@ const AddressPage: React.FC<AddressPageProps> = ({
 									changeAddressButton={i18n.changeAddressButton}
 									findAddressCancelledButton={i18n.findAddressCancelledButton}
 									onFindAddressCancelled={onCancelChanges}
+									onValidatePostcode ={null}
```

Comment:
Once you've made `onValidatePostcode` optional, and none of the cards is using this value, we could safely remove this from the props and not need to pass `null`.